### DISCUSSION TITLE
Ive been hacked bad

### DISCUSSION THREAD

**Discussion by Justintime99er** [community/community#103097] [Other Feature Feedback, Questions, & Ideas]
hello this hacker I think I my neibor they been in my oc my phone's my cameras for over a year and they been filming me as well and now they threaten to post video of me and I they have been making money on me I think the name they are going to be using is  Justintime99er now I don't know his name  because I always kept to myself but I make this person or people stop because there is many and his name I must press charges and have him ,them arrested  .please help me ill give you my full name any info that you may need now what they did without me noing  is go and change all my privacy statements  to make look bad and they did it here as well  .please help me

**Comment by gil-son** [comment]
- You must to contact [Github Support](https://support.github.com/) 
-  Report the situation to your local law enforcement agency. Provide them with all the details you have about the incidents, including the individual's username "Justintime99er" and any other information that might help with the investigation.
- Document all the evidence you have regarding the intrusion and threats. This could include screenshots, messages, or any other information that might be relevant.
- Change passwords for all your accounts, including email, social media, and any other online accounts. Enable two-factor authentication wherever possible.
 - Run antivirus or anti-malware scans on your devices to ensure they are secure.

I hope I was helpfull. Good luck!

**Comment by ossanna16** [comment]
Hi @Justintime99er ,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on your issue. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by ghostinhershell** [comment]
Hi @Justintime99er,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same content in multiple places. As a result, we have deleted the duplicate post, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
